kit/src/lib/Docstring.svelte

```diff
@@ -27,11 +28,13 @@
 			const { name, description } = element;
 			return { ...acc, [name]: description };
 		}, {}) || {};
+	const bgHighlightClass = "bg-yellow-100 dark:bg-[#494a3d]";
```

Comment:
@gary149 wdyt of those highlight values?

Reply:
Those will need to be added on moon-landing right? (for tailwind to take them into account)